nf_core/download.py

```diff
@@ -1695,6 +1723,36 @@ def tidy_tags_and_branches(self):
                 self.retry_setup_local_repo(skip_confirm=True)
                 raise DownloadError(e) from e
 
+    # "Private" method to add the additional custom tags to the repository.
+    def __add_additional_tags(self) -> None:
+        if self.additional_tags:
+            self.ensure_git_user_config(f"nf-core download v{nf_core.__version__}", "core@nf-co.re")
```

Comment:
```suggestion
            self.ensure_git_user_config(f"nf-core download v{nf_core.__version__}", "test@example.com")
```

Reply:
Thanks for the suggestion! I had at the same time googled a bit about suitable sinkhole addresses, and now settled on _dev-null@example.com_. Ultimately, I could not find any information on behalf of ICANN / IETF that one may indeed use any e-mail of that reserved domain for anything else than documentation.